src/main/webapp/app/overview/course-statistics/course-statistics.component.ts

```diff
@@ -45,12 +45,12 @@ export class CourseStatisticsComponent implements OnInit, OnDestroy {
 
     courseId: number;
     private courseExercises: Exercise[];
-    private paramSubscription: Subscription;
+    private paramSubscription?: Subscription;
     private courseUpdatesSubscription: Subscription;
     private translateSubscription: Subscription;
     course?: Course;
 
-    // TODO: improve the types here and use maps instead of java script objects
+    // TODO: improve the types here and use maps instead of java script objects, also avoid any
```

Comment:
I looks like something is missing, I am not sure what should be avoided. Maps?

Reply:
The usage of the "any" type should be avoided. Like in line 82f